packages/ui-components/src/components/DateRangePicker/DateRangePicker.tsx

```diff
@@ -61,16 +61,27 @@ const Label = styled(Button)`
   pointer-events: none;
 `;
 
+interface DateRangePickerProps {
+  startDate?: Moment;
+  endDate?: Moment;
+  minDate?: string;
+  maxDate?: string;
+  granularity?: typeof GRANULARITY_SHAPE;
+  onSetDates?: (startDate: string, endDate: string) => void;
+  isLoading?: boolean;
+  weekDisplayFormat?: string;
```

Comment:
```suggestion
  weekDisplayFormat?: WeekDisplayFormat;
```

There is a WeekDisplayFormat type in the @tupaia/types that we should be able to use

Reply:
Ended up making use of the week display formats from `utils` instead, because `types` didn't expose that type and I don't know enough about the `types` package to move things around